Swift/ReplicatorConfiguration.swift

```diff
@@ -124,69 +124,89 @@ public struct ReplicatorConfiguration {
     public var allowReplicatingInBackground: Bool = false
     #endif
     
+    private var _heartbeat: TimeInterval = 0
     /**
-      The heartbeat interval in second.
-
-      The interval when the replicator sends the ping message to check whether the other peer is still alive.
-      Note: Setting the heartbeat to zero or negative value will result in InvalidArgumentException being thrown.
-      */
-    public var heartbeat: TimeInterval = 300 {
-        willSet(newValue) {
-            guard newValue > 0 else {
+    The heartbeat interval in second.
+    
+    The interval when the replicator sends the ping message to check whether the other peer is still alive. Set the value to
+    zero(by default) means using the default heartbeat of 300 seconds.
+      
+    Note: Setting the heartbeat to negative value will result in InvalidArgumentException being thrown.
+     */
+    public var heartbeat: TimeInterval {
```

Comment:
I feel like this should be an optional type (for any optional configs). But this can be explored and done as a separate ticket.